using_images/s2i_images/dot_net_core.adoc

```diff
@@ -46,7 +46,7 @@ Image stream definitions for the .NET Core on RHEL S2I image are now added
 during {product-title} installations.
 endif::openshift-enterprise[]
 
-The RHEL 7 images are available through Red Hat's subscription registry using:
+The RHEL 7 images are available through Red Hat's subscription registry:
```

Comment:
s/Red Hat's subscription registry/the Red Hat Registry/

See https://access.redhat.com/containers/#/faq.